pkg/losetup/README.md

```diff
@@ -0,0 +1,27 @@
+# LinuxKit losetup
```

Comment:
Could you move this documentation to `./docs` as also @deitch suggested? One reason to have it in `./docs` is that if it stays here and someone fixes, say a typo, then the content hash of the package changes and we'd have to rebuild, although the package itself did not change.